### PR TITLE
Meeting stage view nodejs -> React -> Live share SDK

### PR DESCRIPTION
**Pull request by HunaidHanfee-MSFT**

### CODE REVIEW

**Review by AbhijeetThakur-msft — APPROVED** [review]

File: samples/meetings-stage-view/nodejs/ClientApp/public/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/ClientApp/public/index.html:9]: Can we change content?

File: samples/meetings-stage-view/nodejs/ClientApp/src/components/app-in-meeting.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                // Adding and removing classes based on screen width, to show app in stage view and in side panel
+                $("#todo, #doing, #done").addClass("grid-item");
+                $("#todo, #doing, #done").removeClass("grid-item-sidepanel");
+                $("#boardDiv").addClass("chat-window");
+                $("#boardDiv").removeClass("chat-window-sidepanel");
+            }
+        });
+    }, []);
+
+    // shareToStage.
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/nodejs/ClientApp/src/components/app-in-meeting.jsx:36]: Add proper summary

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and added my comments.

File: samples/meetings-stage-view/nodejs/ClientApp/public/index.html (hunk 1 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/nodejs/ClientApp/public/index.html]: Updated

File: samples/meetings-stage-view/nodejs/ClientApp/src/components/app-in-meeting.jsx (hunk 2 above)
Comment (HunaidHanfee-MSFT) [reply samples/meetings-stage-view/nodejs/ClientApp/src/components/app-in-meeting.jsx]: Added

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**